src/API/Plugins.php

```diff
@@ -431,6 +445,34 @@ public function activate_plugins( $request ) {
 		) );
 	}
 
+	/**
+	 * Return recommend payment plugins.
+	 *
+	 * @param WP_REST_Request $request Full details about the request.
+	 * @return \WP_Error|\WP_HTTP_Response|\WP_REST_Response
+	 */
+	public function recommended_payment_plugin( $request ) {
+		// Default to marketing category (if no category set).
+		$all_plugins   = PaymentPlugins::get_instance()->get_recommended_plugins();
+		$valid_plugins = [];
+		$per_page      = $request->get_param( 'per_page' );
+		$locale        = get_locale();
+
+		foreach ( $all_plugins as $plugin ) {
+			if ( ! PluginsHelper::is_plugin_active( $plugin['product'] ) ) {
+				if ( isset( $plugin['locale-data'] ) && isset( $plugin['locale-data'][ $locale ] ) ) {
+					$locale_plugin = array_merge( $plugin, $plugin['locale-data'][ $locale ] );
+					unset( $locale_plugin['locale-data'] );
+					$valid_plugins[] = $locale_plugin;
```

Comment:
Not used at the moment, but if we decide to support multiple languages, we can easily add it to the `json` now.